config/api.yml

```diff
@@ -42,6 +42,20 @@
     - :subcollection
     :verbs: *g
     :klass: Account
+  :ansible_repositories:
+    :description: Ansible Repositories
```

Comment:
@bdunne sorry for the confusion. Per the [ticket](https://www.pivotaltracker.com/n/projects/1937537/stories/138151635) assigned to me, it was my understanding that this was known as `ansible_repositories`. I should have reached out for more information. 

Also based on that ticket and the specific use case of it, I thought it made sense to make an ansible specific endpoint. 

cc: @h-kataria